tests/ui/regions/multiple-sources-for-outlives-requirement.stderr

```diff
@@ -0,0 +1,20 @@
+error[E0716]: temporary value dropped while borrowed
+  --> $DIR/multiple-sources-for-outlives-requirement.rs:5:38
+   |
+LL | fn foo<'b>() {
+   |        -- lifetime `'b` defined here
+LL |     outlives_indir::<'_, 'b, _>(&mut 1u32);
+   |     ---------------------------------^^^^-- temporary value is freed at the end of this statement
+   |     |                                |
+   |     |                                creates a temporary value which is freed while still in use
+   |     argument requires that borrow lasts for `'b`
+   |
+note: requirements that the value outlives `'b` introduced here
+  --> $DIR/multiple-sources-for-outlives-requirement.rs:1:23
+   |
+LL | fn outlives_indir<'a: 'b, 'b, T: 'a>(_x: T) {}
+   |                       ^^         ^^
```

Comment:
The last commit is independent and changes it to point at *every* requirement, as previously this error was pointing only at the second span, which even though is correct, it isn't very understandable to someone who doesn't have a good grasp of Rust yet. What do you think @lcnr? I can drop the last commit and make it point just at the `'a` here.

Reply:
hmm, I agree that this seems better than just pointing at one of them🤔